src/spaceone/inventory/service/cloud_service_service.py

```diff
@@ -454,101 +462,98 @@ def _make_cloud_service_type_key(resource_data):
     def _make_region_key(resource_data, provider):
         return f'{resource_data["domain_id"]}.{provider}.{resource_data["region_code"]}'
 
-    def _change_metadata_path(self, metadata):
+    def _change_metadata_path(self, metadata, provider=None):
+        if provider is None:
+            provider = 'custom'
+
         plugin_id = self.transaction.get_meta('plugin_id')
 
         if plugin_id:
```

Comment:
change condition
`plugin_id` -> `provider`

Reply:
have reflected thank you.